melissa/config.py

```diff
@@ -0,0 +1,33 @@
+# Melissa
+from profile import data
+import sys
+from tts import TTS
+import importlib
+import pkgutil
+import inspect
+
+dict_tts = dict()
+for (_, name, _) in pkgutil.iter_modules(["tts_engines"]):
+    module = importlib.import_module('.' + name + ".main", "tts_engines")
+    for cls, obj in inspect.getmembers(module):
+        if obj in TTS.__subclasses__():
+            class_ = getattr(module, cls)
+            dict_tts[str(class_.name)] = class_
+print(dict_tts)
+try:
+    if data['tts'] in dict_tts.keys():
+        tts_engine = dict_tts[data['tts']]
+except KeyError as e:
+    print(dict_tts.keys())
```

Comment:
Let's make this error a bit more human-readable. Something like `TTS engine "[whatever the user has specified]" is not available. Available TTS engines: <full list>`.